src/Sfx/package.json

```diff
@@ -39,6 +39,8 @@
     "yargs": "7.0.1"
   },
   "dependencies": {
+    "hawk": "^7.0.7",
+    "hoek": "^5.0.3",
```

Comment:
Please also update the third party notice in standalone as the these dependencies will be included in the standalone release.